package.json

```diff
@@ -39,12 +39,16 @@
   },
   "homepage": "https://github.com/mbebenita/WebAssemblyStudio",
   "dependencies": {
+    "@types/file-saver": "^1.3.0",
+    "@types/jszip": "^3.1.3",
     "@types/minimatch": "^3.0.2",
     "@types/mousetrap": "^1.5.34",
     "@types/react-icons": "^2.2.4",
     "@types/react-modal": "^3.1.1",
     "@types/xterm": "^2.0.3",
     "cassowary": "0.0.2",
+    "file-saver": "^1.3.3",
```

Comment:
Why do we need file-saver?

Reply:
For: [this](https://github.com/haxxorsid/WebAssemblyStudio/blob/0cd09ac9131edf67ddabb7bfb75b8ebf5c08bcbb/src/service.ts#L360)

Reply:
The alternative for generating URL from blob is [URL.createObjectURL()](https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL)